packages/aragon-wrapper/src/rpc/handlers/describe-transaction.js

```diff
@@ -22,6 +22,10 @@ export default async function (request, proxy, wrapper) {
         description: processed.description
       }
     } catch (_) {}
+  } else {
+    return {
+      ...transaction
+    }
```

Comment:
Was trying to encapsulate the logic here: https://github.com/aragon/aragon.js/issues/101#issuecomment-418801302

Though this may not be needed, as if the `evaluatedNotice` is undefined, the `tryEvaluatingRadspec` method returns the fields inside the transaction as well: https://github.com/aragon/aragon.js/blob/1684b7696d48ed3e248ba1dc426e343dff5ed850/packages/aragon-wrapper/src/radspec/index.js#L52

Reply:
Yes, this part isn't needed because we're only interested in returning the `description` or `annotatedDescription` in this API and not the entire transaction (it's already given as input).